cms/tests/placeholder.py

```diff
@@ -306,6 +307,16 @@ def test_placeholder_scanning_nested_super(self):
     def test_placeholder_field_no_related_name(self):
         self.assertRaises(ValueError, PlaceholderField, 'placeholder', related_name='+')
 
+    def test_placeholder_field_db_table(self):
+        """
+        Test for #3891
+        """
+        example = Category.objects.create(
+            name='category',
+            parent=None
+        )
+        self.assertEqual(len(example.description._get_attached_fields()), 1)
```

Comment:
While you're at it, maybe also test that the attached field is the one you expect.